frontend/src/App.js

```diff
@@ -2,7 +2,7 @@ import { createBrowserRouter, RouterProvider } from "react-router-dom";
 import HomePage from "./pages/HomePage";
 import RootLayout from "./pages/RootLayout";
 import EventsPage from "./pages/EventsPage";
-import EventsDetails, {loader as EventsDetailsLoader} from "./pages/EventsDetails";
+import EventsDetails, {loader as EventsDetailsLoader, detailsAction as deleteAction} from "./pages/EventsDetails";
```

Comment:
when using an alias make sure you import the name of original exported function and then set an alias name to it